npe2/manifest/commands.py

```diff
@@ -11,81 +11,81 @@
 
 
 class CommandContribution(BaseModel):
-    """Contribute a command.
-
-    Contribute the UI for a command consisting of a title and (optionally) an
-    icon, category, and enabled state. Enablement is expressed with when
-    clauses. By default, commands show in the Command Palette (⇧⌘P) but they
-    can also show in other menus.
-
-    Presentation of contributed commands depends on the containing menu. The
-    Command Palette, for instance, prefixes commands with their category,
-    allowing for easy grouping. However, the Command Palette doesn't show icons
-    nor disabled commands. The editor context menu, on the other hand, shows
-    disabled items but doesn't show the category label.
-
-    Note: When a command is invoked (from a key binding, from the Command
-    Palette, any other menu, or programmatically), VS Code will emit an
-    activationEvent onCommand:${command}.
+    """Contribute a **command** (a python callable) consisting of a unique `id`,
+    a `title` and (optionally) a `python_path` that points to a fully qualified python
+    callable.  If a `python_path` is not included in the manifest, it *must* be
+    registered during activation with `register_command`.
+
+    Note, some other contributions (e.g. `readers`, `writers` and `widgets`) will
+    *point* to a specific command.  The command itself (i.e. the callable python
+    object) will always appear in the `contributions.commands` section, but those
+    contribution types may add additional contribution-specific metadata.
+
+    ```{admonition} Future Plans
+    Command contributions will eventually include an **icon**, **category**, and
+    **enabled** state. Enablement is expressed with when clauses, that capture a
+    conditional expression determining whether the command should be enabled or not,
+    based on the current state of the program.  (i.e. "*If the active layer is a
+    `Labels` layer*")
+    ```
+
+    Commands will show in a the Command Palette (⇧⌘P) but they can also show in other
+    menus.
```

Comment:
The command palette part should go in the future.  Maybe remove the keyboard shortcut?